bootstrap.sh

```diff
@@ -1,54 +1,67 @@
-#!/bin/bash
+#!/usr/bin/bash
 
-set -eux pipefail
+set -e  # Exit with nonzero exit code if anything fails
```

Comment:
Did this end up working w/ -x?

Reply:
Currently testing in newest push

Reply:
@malexanderboyd still nothing, can you see the travis logs? If not will link it

Reply:
Yes, I can see now - Left comments around this

Reply:
@TzviZeller